Hypriot-OS-Customisation-Instructions.md

```diff
@@ -24,6 +24,47 @@ Insert the card into the micro SD slot on your Raspberry Pi (should work on Rasp
 
 The installation of the OS as described in the previous section should be repeated for the remaining nodes. Make sure to pass the correct hostname (`node-1` ... `node-4`) to the `flash` script and to set the corresponding IP address in the `akka-pi-os.yml` file.
 
+#### Configure password-less login
+
+To prevent logging in on each pi node we will set-up password-less login. We assume that we want to log in to account _akkapi_ on a pi from an account (_userxxx_) on your laptop.
+
+##### Prerequisite
+In order to setup password-less login, you first need a pair of authentication keys on the laptop. If already have these keys, you can proceed to setting up the login. If this isn't the case, generate them:
+
+```
+$ ssh-keygen -t rsa
+Generating public/private rsa key pair.
+Enter file in which to save the key (/home/userxxx/.ssh/id_rsa): 
+Created directory '/home/userxxx/.ssh'.
+Enter passphrase (empty for no passphrase): 
+Enter same passphrase again: 
+Your identification has been saved in /home/userxxx/.ssh/id_rsa.
+Your public key has been saved in /home/userxxx/.ssh/id_rsa.pub.
+The key fingerprint is:
+3e:4f:05:79:3a:9f:96:7c:3b:ad:e9:58:37:bc:37:e4 userxxx@A
+``` 
+
+Use an empty passphrase (just hit _Enter_)
+
+As can be seen from the output generated by _ssh-keygen_, a pair of files (`id_rsa` and `id_rsa.pub`) are created in the (hidden) folder `.ssh` in the users home folder.
+
+##### Setup Steps
+1. create a `.ssh` folder on the _akkapi_'s home folder on the Pi and copy the public key (`id_rsa.pub`) to a file named `authorized_keys` in the `.ssh` folder. You will have to supply the password for the _akkapi_ account for the two commands launched.
+
+    ```
+    [userxxx@Eric-Loots-MBP] $ for node in 0 1 2 3 4;do ssh akkapi@node-${node} mkdir .ssh; done
+    akkapi@node-0's password:
+    ```
+2. Copy the contents of your *public key* to the `authorized_keys` file:
+
+     ```
+     [userxxx@Eric-Loots-MBP] $ for node in 0 1 2 3 4;do cat .ssh/id_rsa.pub | ssh akkapi@node-${node} 'cat >> .ssh/authorized_keys'; done
```

Comment:
I would change the cat command to point to the `.ssh` folder in the user's home directory. i.e:
`for node in 0 1 2 3 4;do cat ~/.ssh/id_rsa.pub | ssh akkapi@node-${node} 'cat >> .ssh/authorized_keys'; done`

Reply:
@kikiya This change (adding `~/` in front of `.ssh/id_rsa.pub` was missed in your update to this PR. Maybe add that to a new PR?